### PR TITLE
Add 3 pass version check

### PR DESCRIPTION
**Pull request by haozech**
### PR types
New features

### PR changes
OPs

### Describe
Add three passes' version check

### CODE REVIEW

**Comment by haozech** [comment]
@shangzhizhou  辛苦review下，三个pass的单测都已存在：
[`python/paddle/fluid/tests/unittests/ir/inference/test_conv_elementwise_add_fuse_pass.py `](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/tests/unittests/ir/inference/test_conv_elementwise_add_fuse_pass.py)

[`python/paddle/fluid/tests/unittests/ir/inference/test_conv_elementwise_add_act_fuse_pass.py `](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/tests/unittests/ir/inference/test_conv_elementwise_add_act_fuse_pass.py)

[`python/paddle/fluid/tests/unittests/ir/inference/test_conv_elementwise_add2_act_fuse_pass.py`](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/tests/unittests/ir/inference/test_conv_elementwise_add2_act_fuse_pass.py)

**Review by shangzhizhou — COMMENTED** [review]
在确认op属性都支持的情况下，请在单测中添加版本检测用例，可参考卫波已合入PR

File: python/paddle/fluid/tests/unittests/ir/inference/test_conv_elementwise_add2_act_fuse_pass.py [hunk 1]
```diff
@@ -46,6 +47,7 @@ def test_check_output(self):
         if core.is_compiled_with_cuda():
             use_gpu = True
             self.check_output_with_option(use_gpu)
+        PassVersionChecker.IsCompatible('conv_elementwise_add2_act_fuse_pass')
```
Comment (shangzhizhou) [inline python/paddle/fluid/tests/unittests/ir/inference/test_conv_elementwise_add2_act_fuse_pass.py]: 需要添加返回值断言
`self.assertTrue(...)`

File: python/paddle/fluid/tests/unittests/ir/inference/test_conv_elementwise_add_act_fuse_pass.py [hunk 2]
```diff
@@ -48,6 +49,7 @@ def test_check_output(self):
         if core.is_compiled_with_cuda():
             use_gpu = True
             self.check_output_with_option(use_gpu)
+        PassVersionChecker.IsCompatible('conv_elementwise_add_act_fuse_pass')
```
Comment (shangzhizhou) [inline python/paddle/fluid/tests/unittests/ir/inference/test_conv_elementwise_add_act_fuse_pass.py]: 同上

File: python/paddle/fluid/tests/unittests/ir/inference/test_conv_elementwise_add_fuse_pass.py [hunk 3]
```diff
@@ -44,6 +45,7 @@ def test_check_output(self):
         if core.is_compiled_with_cuda():
             use_gpu = True
             self.check_output_with_option(use_gpu)
+        PassVersionChecker.IsCompatible('conv_elementwise_add_fuse_pass')
```
Comment (shangzhizhou) [inline python/paddle/fluid/tests/unittests/ir/inference/test_conv_elementwise_add_fuse_pass.py]: 同上

Comment (haozech) [reply python/paddle/fluid/tests/unittests/ir/inference/test_conv_elementwise_add_fuse_pass.py]: Done, thanks

File: python/paddle/fluid/tests/unittests/ir/inference/test_conv_elementwise_add_act_fuse_pass.py (hunk 2 above)
Comment (haozech) [reply python/paddle/fluid/tests/unittests/ir/inference/test_conv_elementwise_add_act_fuse_pass.py]: Done, thanks

File: python/paddle/fluid/tests/unittests/ir/inference/test_conv_elementwise_add2_act_fuse_pass.py (hunk 1 above)
Comment (haozech) [reply python/paddle/fluid/tests/unittests/ir/inference/test_conv_elementwise_add2_act_fuse_pass.py]: Done, thanks

**Review by shangzhizhou — APPROVED** [review]
LGTM

**Comment by haozech** [comment]
@Superjomn 辛苦review下

**Review by Superjomn — APPROVED** [review]
LGTM

**Review by Superjomn — APPROVED** [review]
LGTM

**Review by Superjomn — COMMENTED** [review]
需要增加精度测试

**Pull request merged**